### PR TITLE
Adding docs for Migration Readiness Report

### PR DESCRIPTION
**Pull request by SaschaPeukert**
[Trello](https://trello.com/c/2a8ZbFKR/2525-document-and-publish-mrr-page-in-upx-remove-ft)

Merging this will wait until PM is ready to release

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-507.surge.sh

New pages:
- [Migration Readiness Report](https://neo4j-docs-aura-507.surge.sh/aura/auradb/managing-databases/migration-readiness)

File: modules/ROOT/pages/auradb/managing-databases/migration-readiness.adoc [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+image::mrr-show-query-log-button.png[width=400]
+
+== Deprecated driver usage
+
+image::mrr-fetch-driver-stats.png[width=400]
+
+After selecting a time frame of at most 24 hours, you can click the "Fetch driver statistics" button in this section.
+By default, the filters in the popup are set to show only driver usage with potential issues in any database, including the system database.
+You can change those freely to see all driver usage, e.g.
```
Comment (AlexicaWright) [inline modules/ROOT/pages/auradb/managing-databases/migration-readiness.adoc:81]:
```suggestion
You can change those freely to see all driver usage, for example.
```

File: modules/ROOT/pages/auradb/managing-databases/migration-readiness.adoc [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+Clicking on the "Fetch" button in the popup will populate the report's table with the applicable data.
```
Comment (AlexicaWright) [inline modules/ROOT/pages/auradb/managing-databases/migration-readiness.adoc:83]:
```suggestion
Use the button in the popup to fetch applicable data to populate the report's table.
```

File: modules/ROOT/pages/auradb/managing-databases/migration-readiness.adoc [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+Depending on the type of client accessing the Neo4j database, links are provided in the column “Latest version” to help with the upgrade.
+
+image::mrr-driver-table.png[]
+
+Like the Cypher deprecations table, the last column links to a view of this specific driver's executed queries in the Aura Query Log tool.
+The tool can provide information on each query execution in which the selected driver was used.
+The tool can only view queries on non-system databases.
```
Comment (AlexicaWright) [inline modules/ROOT/pages/auradb/managing-databases/migration-readiness.adoc:90]:
```suggestion
The tool can view queries on all databases except the `system` database.
```

File: modules/ROOT/pages/auradb/managing-databases/migration-readiness.adoc [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+
+This section provides information on how to deal with deprecated indexes that may be used in version 4 but need to be handled before or while moving to version 5.
+
+This part involves manually running a provided Cypher query on your database to identify the deprecated indexes and then deciding how to best deal with them.
+Further enhancements to this feature will be provided in the future.
+
+=== Next steps
+
+After implementing all the recommended fixes from the report, you can now test the migration.
+To do that, please follow the steps outlined in the docs after pressing the "Test migration" button at the bottom of the page.
```
Comment (AlexicaWright) [inline modules/ROOT/pages/auradb/managing-databases/migration-readiness.adoc:102]:
```suggestion
To do that, follow the steps outlined in the docs after pressing the "Test migration" button at the bottom of the page.
```
Which docs are you referring to?

**Review by AlexicaWright — COMMENTED** [review]
Some editorial comments.

File: modules/ROOT/pages/auradb/managing-databases/migration-readiness.adoc [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+The last column in the table of Cypher deprecations links to a view of this specific query in the Aura Query Log Analyzer tool, which can provide information on each execution of the selected query.
+The tool can only view queries on non-system databases.
+
+image::mrr-show-query-log-button.png[width=400]
+
+== Deprecated driver usage
+
+image::mrr-fetch-driver-stats.png[width=400]
+
+After selecting a time frame of at most 24 hours, you can click the "Fetch driver statistics" button in this section.
```
[the remark this answers is not in the document]
Comment (SaschaPeukert) [reply modules/ROOT/pages/auradb/managing-databases/migration-readiness.adoc:79]: You sure about the end with ``` ?

File: modules/ROOT/pages/auradb/managing-databases/migration-readiness.adoc (hunk 4 above)
Comment (SaschaPeukert) [reply modules/ROOT/pages/auradb/managing-databases/migration-readiness.adoc:102]: The docs that are linked directly from the MRR page. So far its this link: https://neo4j.com/docs/aura/tutorials/upgrade/#_perform_the_upgrade

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-507.surge.sh

New pages:
- [Migration Readiness Report](https://neo4j-docs-aura-507.surge.sh/aura/auradb/managing-databases/migration-readiness)

File: modules/ROOT/pages/auradb/managing-databases/migration-readiness.adoc (hunk 5 above)
[the remark this answers is not in the document]
Comment (AlexicaWright) [reply modules/ROOT/pages/auradb/managing-databases/migration-readiness.adoc:81]: No idea where those came from!

File: modules/ROOT/pages/auradb/managing-databases/migration-readiness.adoc (hunk 4 above)
Comment (AlexicaWright) [reply modules/ROOT/pages/auradb/managing-databases/migration-readiness.adoc:104]: Alright, in that case, I suggest:
_Use the "Test migration" button at the bottom of the page and then follow the steps outlined in the docs._

File: modules/ROOT/pages/auradb/managing-databases/migration-readiness.adoc [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+The last column in the table of Cypher deprecations links to a view of this specific query in the Aura Query Log Analyzer tool, which can provide information on each execution of the selected query.
+The tool can view queries on all databases except the `system` database.
+
+image::mrr-show-query-log-button.png[width=400]
+
+== Deprecated driver usage
+
+image::mrr-fetch-driver-stats.png[width=400]
+
+After selecting a time frame of at most 24 hours, you can click the "Fetch driver statistics" button in this section.
```
Comment (SaschaPeukert) [inline modules/ROOT/pages/auradb/managing-databases/migration-readiness.adoc:81]:
```suggestion
After selecting a time frame of a maximum of 24 hours, use the button to fetch the driver statistics in this section.
```

File: modules/ROOT/pages/auradb/managing-databases/migration-readiness.adoc [hunk 7]
```diff
@@ -0,0 +1,104 @@
...
+
+This section provides information on how to deal with deprecated indexes that may be used in version 4 but need to be handled before or while moving to version 5.
+
+This part involves manually running a provided Cypher query on your database to identify the deprecated indexes and then deciding how to best deal with them.
+Further enhancements to this feature will be provided in the future.
+
+=== Next steps
+
+After implementing all the recommended fixes from the report, you can now test the migration.
+To do that, please follow the steps outlined in the docs after pressing the "Test migration" button at the bottom of the page.
```
Comment (SaschaPeukert) [inline modules/ROOT/pages/auradb/managing-databases/migration-readiness.adoc:104]:
```suggestion
Use the "Test migration" button at the bottom of the page and then follow the steps outlined in the docs.
```

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-507.surge.sh

New pages:
- [Migration Readiness Report](https://neo4j-docs-aura-507.surge.sh/aura/auradb/managing-databases/migration-readiness)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-507.surge.sh

New pages:
- [Migration Readiness Report](https://neo4j-docs-aura-507.surge.sh/aura/auradb/managing-databases/migration-readiness)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-507.surge.sh

New pages:
- [Migration Readiness Report](https://neo4j-docs-aura-507.surge.sh/aura/auradb/managing-databases/migration-readiness)

**Pull request merged**